src/v2/controllers/vreplicaset_controller/proof/helper_invariants/predicate.rs

```diff
@@ -102,19 +102,16 @@ pub open spec fn garbage_collector_does_not_delete_vrs_pods(vrs: VReplicaSetView
             &&& msg.dst.is_APIServer()
             &&& msg.content.is_APIRequest()
         } ==> {
-            let req = msg.content.get_delete_request(); 
-            &&& msg.content.is_delete_request()
-            &&& req.preconditions.is_Some()
-            &&& req.preconditions.unwrap().uid.is_Some()
-            &&& req.preconditions.unwrap().uid.unwrap() < s.api_server.uid_counter
+            let req = msg.content.get_get_then_delete_request(); 
```

Comment:
The garbage collector does not use the new API; it sends good old-fashioned delete requests.